style.css

```diff
@@ -503,6 +521,17 @@ h1 {
     align-self: self-start;
   }
 
+  .name-group {
```

Comment:
- [ ] While you make a great effort 😃 I see that you make the font size of the text here greater than the font size as you see here in your case you put the font size ```40px``` , but if you take a look at the Figma Design you will see that the font size is ```30px``` , so let's change it to make the text like the Figma Design 😊💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/da0a48fa-eca4-4bb5-9c51-8e21d10e1a52)  |    ![image](https://github.com/AsumVictor/Porfolio/assets/83360636/8383a872-b7f0-4449-887f-536d7b4ce3df)  |